wdl2cwl/tests/cwl_files/InternalTasks.cwl

```diff
@@ -67,8 +67,8 @@ $graph:
                   --HAPLOTYPE_MAP $(inputs.haplotype_database_file.path) \
                   --EXPECTED_GENOTYPING_PLATFORMS FLUIDIGM \
                   --EXPECTED_GENOTYPING_PLATFORMS GENERAL_ARRAY \
-                  $(inputs.ignoreSpecificGenotypesLsid ? inputs.ignoreSpecificGenotypesLsid === null ? "" : "--IGNORE_SPECIFIC_GENOTYPES_LSID \"" + inputs.ignoreSpecificGenotypesLsid + "\"" : "") \
-                  $(inputs.ignoreSpecificGenotypesPlatform ? inputs.ignoreSpecificGenotypesPlatform === null ? "" : "--IGNORE_SPECIFIC_GENOTYPES_PLATFORM \"" + inputs.ignoreSpecificGenotypesPlatform + "\"" : "") \
+                  $(inputs.ignoreSpecificGenotypesLsid !== null ? inputs.ignoreSpecificGenotypesLsid === null ? "" : "--IGNORE_SPECIFIC_GENOTYPES_LSID \"" + inputs.ignoreSpecificGenotypesLsid + "\"" : "") \
+                  $(inputs.ignoreSpecificGenotypesPlatform !== null ? inputs.ignoreSpecificGenotypesPlatform === null ? "" : "--IGNORE_SPECIFIC_GENOTYPES_PLATFORM \"" + inputs.ignoreSpecificGenotypesPlatform + "\"" : "") \
```

Comment:
a bit redundant, yes, just like the original WDL (they don't need `defined()`
https://github.com/common-workflow-lab/wdl-cwl-translator/blob/63d60b51421a68b819411e36ed47a1df547db411/wdl2cwl/tests/wdl_files/InternalTasks.wdl#L96

Reply:
This looks very interesting. So each time the `defined( )` function is used it will checked to see if it's `null` how is it different from when it's used in placeholder options? Do you have a link to a wdl file that has this functionality?